lib/net/pool.js

```diff
@@ -1291,7 +1419,7 @@ class Pool extends EventEmitter {
     }
 
     // We want compact blocks!
-    if (this.options.compact)
+    if (this.options.compact && !this.options.neutrino)
```

Comment:
Maybe revert this change this was fixed by setting the compact option to false by default

Reply:
Right